api/lib/rules-next-id

```diff
@@ -30,11 +30,23 @@ use esmith::ConfigDB;
 use List::Util qw[max];
 
 my $db = esmith::ConfigDB->open('fwrules');
+my $sdb = esmith::ConfigDB->open('separators');
 
-my $max_id = 0;
+my $max_id1 = 0;
```

Comment:
Can we use a better variable naming?
Like:
- max_id1 -> max_rule_id
- max_id2 -> max_sep_id

Reply:
done